docs/webiny-apps/headless-cms/features/device-frontend-agnostic.md

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
I don't think this was added to the `sidebars.js` codepiece since this doesn't show up on the netlify link

Reply:
We will see if we add that as a page, or just use the Headless CMS + Gatsby tutorial.